packages/data/src/plugins/async-generator/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { applyMiddleware } from 'redux';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import asyncGeneratorMiddleware, { toAsyncIterable } from '../../middlewares/async-generator';
```

Comment:
Minor: To me, it seems more important to keep plugins self-contained than to keep middlewares together, i.e. middleware defined within `src/plugins/async-generator`

Reply:
Where would you put the "promise" middleware?

Reply:
I don't really feel strongly where, either inline at `async-generator/index.js` or a separate `async-generator/middleware.js` file.

Reply:
The "promise" middleware is not a plugin middleware it's "core". It will be kept even if the async-generator plugin is removed.